engine/classes/Elgg/Menu/MenuItems.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Elgg\Menu;
+
+use Elgg\Collections\Collection;
+
+/**
+ * A collection of menu items
+ */
+class MenuItems extends Collection {
```

Comment:
wouldn't it make sense to overrule the `__construct()` and default the `$item_class` to `ElggMenuItem::class`?? It could save some weird constructor calls below

Reply:
Sure.

Reply:
Done